### DISCUSSION TITLE
Podman Built Image error can't be deployed on AWS ECS, does ECS isn't OCI Compliant?

### DISCUSSION THREAD

**Discussion by benyaminl** [containers/podman#21238] [Q&A] [answered]
Hello, I have very strange condition where image that I build using buildah/podman have bigger size, and when deployed on ECS it failed to start. 

Before pushing to the AWS container registry, I already try to rerun and it's okay, and I even try to rerun the image on docker it's also okay, but it's broken on AWS, and only show error failed to start... which is strange. 

My Container is very simple container, that build for ASP.NET Core 

```
#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.

FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS base
WORKDIR /app
EXPOSE 80


FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster AS build
WORKDIR /src
COPY . .

FROM build AS publish
RUN dotnet publish "project.csproj" -c Release -o /app/publish

FROM base AS final
WORKDIR /app
COPY --from=publish /app/publish .
ENTRYPOINT ["dotnet", "thedll.dll"]
```

Is ECS really not supporting container that built using podman or buildah? Or There are something that I need check regarding it? 

Thank you

**Comment by rhatdan** [comment, marked as answer]
Try building and pushing with the --format docker.  Podman/Buildah default to OCI Format, perhaps ECS is chocking on this.

**Reply by benyaminl** [reply]
Thanks Mr. Walsh. I will try this. Thank you !

**Reply by benyaminl** [reply]
Yeah this help with buildah. Thanks!

**Reply by rhatdan** [reply]
Complain to ECS that they should support OCI Images, the standard has been out there for years.

**Reply by neurostacks** [reply]
I hope AWS can support the standards. The worst part is it fails without info—just a generic message about the container not starting, with no logs or error details, not even CloudTrail can trace. Nonetheless, I stuck with Podman, and using --format saved my time. Thanks for this thread.
